packages/webdriverio/src/index.ts

```diff
@@ -48,7 +46,7 @@ export default class AxeBuilder {
     // Treat everything as Browser<'async'>:
     // - Anything sync can also run async, since JS can await sync functions
     // - Ignore MultiRemoteBrowser, which is just Browser with extra props
-    this.client = client as Browser<'async'>;
+    this.client = client as Browser;
```

Comment:
If we are casting `client as Browser` then the type: 

```ts
WdioBrowser = Browser | MultiRemoteBrowser;
```

Could just be `import type { Browser } from 'webdriverio` and we could simply remove `WdioBrowser`. Otherwise we could remove the casting and let TypeScript handle whether the user passes in a type of `Browser` or `MultiRemoveBrowser`